code/game/machinery/vending.dm

```diff
@@ -1561,6 +1561,53 @@
 	idle_power_usage = 211
 	vendor_department = DEPARTMENT_OFFSHIP
 
+/obj/machinery/vending/style
+	name = "Style vendor"
+	desc = "Vendor selling stolen stylish clothing."
+	product_slogans = "Highly stylish clothing for sale!;Latest fashion trends right here!"
+	product_ads = "Contact our consumer department for refunds, fees apply.;Coolness sold here."
+	icon_state = "style"
+	products = list(
+		/obj/item/clothing/gloves/knuckles = 3,
+		/obj/item/clothing/head/ranger = 4,
+		/obj/item/clothing/head/inhaler = 2,
+		/obj/item/clothing/shoes/redboot = 4,
+		/obj/item/clothing/shoes/jackboots/longboot = 3,
+		/obj/item/clothing/under/white = 4,
+		/obj/item/clothing/under/red = 4,
+		/obj/item/clothing/under/green = 4,
+		/obj/item/clothing/under/grey = 4,
+		/obj/item/clothing/under/black = 4,
+		/obj/item/clothing/under/dress = 4,
+		/obj/item/clothing/under/dress/white = 4,
+		/obj/item/clothing/under/helltaker = 4,
+		/obj/item/clothing/under/johnny = 3,
+		/obj/item/clothing/under/raider = 3,
+		/obj/item/clothing/suit/storage/triad = 2,
+		/obj/item/clothing/suit/storage/akira = 2,
+					)
+	prices = list(
+		/obj/item/clothing/gloves/knuckles = 650,
+		/obj/item/clothing/head/ranger = 550,
+		/obj/item/clothing/head/inhaler = 750,
+		/obj/item/clothing/shoes/redboot = 450,
+		/obj/item/clothing/shoes/jackboots/longboot = 550,
+		/obj/item/clothing/under/white = 600,
+		/obj/item/clothing/under/red = 600,
+		/obj/item/clothing/under/green = 600,
+		/obj/item/clothing/under/grey = 600,
+		/obj/item/clothing/under/black = 600,
+		/obj/item/clothing/under/dress = 600,
+		/obj/item/clothing/under/dress/white = 600,
+		/obj/item/clothing/under/helltaker = 600,
+		/obj/item/clothing/under/johnny = 750,
+		/obj/item/clothing/under/raider = 750,
+		/obj/item/clothing/suit/storage/triad = 1200,
+		/obj/item/clothing/suit/storage/akira = 750,
+
+					)
+	vendor_department = DEPARTMENT_GUILD
```

Comment:
You might not actually need this since all vendors default to guild. Double check though

Reply:
![image](https://user-images.githubusercontent.com/59490776/121774908-f1ee9b00-cb84-11eb-8b57-4cfad1592665.png)

Reply:
yeah, they do